pkg/logging/logging.go

```diff
@@ -44,6 +44,18 @@ type Driver interface {
 
 type DriverFactory func(map[string]string) (Driver, error)
 
+func ValidateLogOpts(logDriver string, logOpts map[string]string) error {
+	switch logDriver {
+	case "json-file":
+		return JSONFileLogOptsValidate(logOpts)
+	case "journald":
+		return JournalLogOptsValidate(logOpts)
+	case "fluentd":
+		return FluentdLogOptsValidate(logOpts)
+	}
```

Comment:
Driver names shouldn’t be hard coded

see https://github.com/containerd/nerdctl/blob/3f25af7dc8e3843641fce3b1a586c14d6f7aedff/pkg/logging/logging.go#L49